packages/oui-skeleton/README.md

```diff
@@ -0,0 +1,26 @@
+# Skeleton
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+### Default
+```html:preview
+<oui-skeleton></oui-skeleton>
+```
+
+### Size
```

Comment:
Maybe size is not the thing we want.
Typically the skeleton graphically represents a text we don't know in advance. So the size should be... randomized :)
Maybe the right parameters here should be "min-width" and eventually "width" / "auto" if we want to force the width... ?
@AxelPeter what do you think about?

Reply:
Nice idea. But I don't think it's the case for all uses.
We can even have both with an attribute "randomized" for example :)

In fact, I'm not a fan of replacing `style="width: 80%"` just by `width="80%"`. That's why by default, I let the skeleton with `auto` to let the user set its width.

I'll check to add your idea